pages/_index.md

```diff
@@ -2,35 +2,36 @@
 
 2021-02-01: Draft 2020-12 has been published!
 
-The IETF document IDs are of the form `draft-bhutton-*-00`.
+The document IDs are of the form `draft-bhutton-*-00`.
 
 We are using dates for meta-schemas, which are what implementations should use to determine behavior,
 so we will usually refer to `2020-12` (without the word "draft") on this web site.
 
-See the [Specification page](/specification) for details about naming and numbering.
+See the [Specification Links](/specification-links) page for details about naming and numbering.
 
 ### The Path to Standardization
 
-The JSON Schema project intends to shepherd all three draft series to either: RFC status, the equivalent within another standards body, and/or join a foundation and establish self publication rules.
+The JSON Schema project intends to self-publish the two JSON Schema specifications with the OpenJS Foundation. For the Relative JSON Pointer specification, we will likely pursue an RFC with IETF.
```

Comment:
Henry is actively pursuing the RJP spec with IETF.  Not sure if that means tweaking this.

Reply:
I think it's fine. RJP hasn't yet been adopted by an IETF working group. If you want to propose alternate wording, I'd be happy to include it.